### ISSUE TITLE
number of approvals

### ISSUE THREAD

**Issue by damdam-s** [OCA/oca-github-bot#18]
Following discussion in #1 

Concerning the number of approvals needed, maybe we should allow a per-repo configuration (in a file or whatever) with a fallback value set to 2.

Any opinions?

**Comment by pedrobaeza** [comment]
I don't think so. If you are PSC, you have permissions for merging, so you can do it in any moment without the number of approvals.

**Comment by legalsylvain** [comment]
AFAIK, it's not per repo, but per maturity module. I read somewhere that if module was flagged as 'beta', less approvals was required. It isn't ?
